drivers/sdhc/sdhc_esp32.c

```diff
@@ -1444,11 +1444,10 @@ static DEVICE_API(sdhc, sdhc_api) = {
 					.ddr50_support = false,                                    \
 					.sdr104_support = false,                                   \
 					.sdr50_support = false,                                    \
-					.bus_8_bit_support = false,                                \
-					.bus_4_bit_support =                                       \
-						(DT_INST_PROP(n, bus_width) == 4) ? true : false,  \
-					.hs200_support = false,                                    \
-					.hs400_support = false}}};                                 \
+					.bus_8_bit_support = false},                               \
+			  .bus_4_bit_support = (DT_INST_PROP(n, bus_width) == 4) ? true : false,   \
```

Comment:
```suggestion
			  .bus_4_bit_support = (DT_INST_PROP(n, bus_width) == 4),   \
```
is already a boolean, no need for cond operator